build.gradle

```diff
@@ -74,6 +74,10 @@ dependencies {
     detektPlugins libs.detekt.formatting
 }
 
+application {
+    mainClassName = 'com.exactpro.th2.infraoperator.Th2CrdController'
```

Comment:
issue(non-blocking): this property is deprecated. Please, use `mainClass.set()` instead